src/main/resources/application.properties

```diff
@@ -2,5 +2,5 @@ spring.h2.console.enabled=true
 spring.h2.console.path=/h2-console
 spring.jpa.hibernate.ddl-auto=create
 spring.datasource.url=jdbc:h2:mem:costumes
-spring.thymeleaf.mode=LEGACYHTML5
+spring.thymeleaf.mode=HTML
```

Comment:
czemu tak?

Reply:
Przy uruchomieniu projektu, Spring wyświetla ostrzeżenie "Template Mode 'LEGACYHTML5' is deprecated. Using Template Mode 'HTML' instead".  Najprawdopodobniej wynika to z faktu, że w pom nie mamy wskazanej wersji thymeleaf'a, a więc używamy najnowszej (?).  Z tego co wyczytałem w najnowszej wersji html 5 został zastąpiony przez html i stąd to ostrzeżenie. Natomiast obie wersje tego ustawienia są obsługiwane i nie wpływa to na działanie.   Którą wersję radziłbyś zostwić?

Reply:
Na chwilę obecną pozostawiamy  mode = LEGACYHTML5.